RELEASE_NOTES.md

```diff
@@ -1,6 +1,18 @@
 Release Notes
 =============
 
+v1.0.0-alpha.xx
+---------------
+
+### Improvements
+
+- Attempting to retrieve a trait property with
```

Comment:
[consider] Could view this as a breaking change, if people are already relying on exceptions to tell them if a trait is not imbued.

Reply:
Good call - updated [here](https://github.com/OpenAssetIO/OpenAssetIO/pull/1081/commits/bd811520f1a4d09b0d3cb6e288d0372448ddcbe9)